### DISCUSSION TITLE
Date / Time output in logs

### DISCUSSION THREAD

**Discussion by randy-cruz** [isid0re/SoloLeveling#325] [Ideas]
Please add date / time to Script and Leveling performance logs. Similar to kolbot/d2bs/logs: 

[08/06/21 00:00:04] D2BS 32800: ÿc7Picked up ÿc4322 Gold

Would be helpful to those of us that want to dig deeper into script performance or look back on when events took place. I'm playing with graphing bot performance over time and this would be a godsend.

**Comment by isid0re** [comment]
I believe I understand your interest in using the standard output as it is simple to follow from a clock standpoint with Excel. The timer issue of using the stock method would affect actual time tracking of ingame scripts/leveling moments. Specifically, the tracking of time between leveling (Split Time in the LevelingPerformance) and script (Sequence Time in ScriptPerformance). The tick counts stamped are the times from start to stop of each event. 

I use the CSV files to track run times and performance for leveling and progression through the various areas and scripts by importing into excel. I analyze them by running pivot tables and graphs on the datasets.

If you are able to join the discord community and reach out to me in a PM and we can further the conversation, so I can have a clearer idea on why you would need the logging that way. If the use case is there I'm open to adjusting it.

thanks
-isi
